src/OrbitPaths/Paths.cpp

```diff
@@ -22,6 +25,8 @@
 
 ABSL_FLAG(std::string, log_dir, "", "Set directory for the log.");
 
+constexpr std::string_view kOrbitFolderName{"Orbit"};
```

Comment:
I think I would prefer `kOrbitFolderInDocumentsName` for extra clarity.

Reply:
done